src/libraries/System.Net.WebSockets.Client/src/Resources/Strings.resx

```diff
@@ -78,6 +78,9 @@
   <data name="net_WebSockets_Connect101Expected" xml:space="preserve">
     <value>The server returned status code '{0}' when status code '101' was expected.</value>
   </data>
+  <data name="net_WebSockets_Connect200Expected" xml:space="preserve">
+    <value>The server returned status code '{0}' when status code '200' was expected.</value>
+  </data>
```

Comment:
Instead of adding an additional message, can we just change the existing one to also parameterize the expected code?